python/README.md

```diff
@@ -4,14 +4,45 @@ This folder contains sample applications written in Python Programming Language
 These applications have been tested with python 2.7 and 3.0 versions on Mac OS X.
 
 ## iot-rest-api-sample.py
-This sample app creates one event each of type - custom, network and error events.
+This sample app creates and sends one event each of type - custom, network and error events.
 
-Below variables need to be set before running the application.
-```python
-iot[appKey]  //AppKey generated in the controller when creating new application
-iot[collectorUrl] //URL to validate or send the events to collector
+Below is usage information for the sample app. Provide `appkey` as an input along with any options as needed.
+
+```sh
+USAGE: python iot-rest-api-sample.py appkey [options]
+options:
+-c, --collectorurl <url> IoT Collector URL to which the beacons should be sent to.
+-u, --url <url>          URL to trigger network request and capture network event.
+-x, --request <command>  Specify the request method to url. It is set to GET by default.
+-d, --data <data>        Data in JSON format that is to be sent in a POST request.
+-v, --verbose            Enable Debug Information. It is disabled by default.
+-h, --help               Display available options
+```
+
+Below are the default values used by the sample app if above options are not given
+```sh
+DEFAULT PARAMS:
+collectorurl = https://iot-col.eum-appdynamics.com
+```
+
+Here are some examples of using the sample app:
+
+Display Usage Information
+```sh
+$ python iot-rest-api-sample.py -h
 ```
-Make sure all the packages imported in the file are installed.
+
+Send Sample Custom, Network, and Error Events to default APPD Collector
+```sh
+$ python iot-rest-api-sample.py <appkey>
 ```
-$ python iot-rest-api-sample.py
+
+Send Sample Custom, Network, and Error Events to Custom Collector (http://localhost:9001)
+```sh
+$  python iot-rest-api-sample.py <appkey> -c http://localhost:9001
+```
+
+Trigger POST Network Request to URL (http://yoururl.com) with data in JSON format. Capture and Send Network Event to Custom Collector (http://localhost:9001)
+```sh
+$  python iot-rest-api-sample.py <appkey> -c http://localhost:9001 -u http://yoururl.com -x POST -d '{"param1"="value1"}'
```

Comment:
`-X`

Reply:
DONE

Reply:
will deal with URL splitting in a separate ticket.